src/components/garden_admin_card.tsx

```diff
@@ -1,83 +1,146 @@
 import {
   AppBar,
+  Box,
   Button,
   Card,
   CardActions,
   CardContent,
+  Grid,
+  Modal,
   Toolbar,
   Typography,
 } from '@mui/material'
-import Table from 'components/table'
+import { Divider } from 'components/Divider'
+import { GardenStatusAlert } from 'components/GardenStatusAlert'
 import useGardens from 'hooks/useGardens'
+import { Dispatch, SetStateAction, useState } from 'react'
 import { Link as RouterLink } from 'react-router-dom'
 import { Garden } from 'types/backend-types'
-import { TableState } from 'types/custom-types'
+import { SnackbarState } from 'types/custom-types'
 
 interface GardenAdminCardProps {
   garden: Garden
+  setRequestStatus: Dispatch<SetStateAction<SnackbarState | undefined>>
 }
 
-const GardenAdminCard = ({ garden }: GardenAdminCardProps) => {
+const GardenAdminCard = ({
+  garden,
+  setRequestStatus,
+}: GardenAdminCardProps) => {
+  const [open, setOpen] = useState(false)
+
   const { deleteGarden } = useGardens()
-  function getTableData() {
-    return [
-      ['Status', garden.status],
-      ['Namespaces', garden.namespaces.length],
-      ['Systems', garden.systems.length],
-    ]
-  }
 
-  const state: TableState = {
-    formatData: getTableData,
-    tableHeads: [],
-    includePageNav: false,
-    disableSearch: true,
+  const style = {
+    position: 'absolute' as const,
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    bgcolor: 'background.paper',
+    boxShadow: 24,
+    p: 4,
+    height: 'auto',
+    overflow: 'auto',
   }
 
-  function getDeleteButton(connection_type: string) {
-    if (connection_type !== 'LOCAL') {
-      return (
-        <Button
-          onClick={() => deleteGarden(garden.name)}
-          variant="contained"
-          color="secondary"
-        >
-          Delete {garden.name}
-        </Button>
-      )
-    }
-  }
+  const handleDeleteClick = () => {
+    deleteGarden(garden.name)
+      .then(() => {
+        setRequestStatus({
```

Comment:
It would be ideal if this also refreshed the garden list. Please open a ticket to refactor this so that the lists of gardens on the garden management page is refreshed only if the 'confirm' button is pushed on the modal popup and the API call returns without error.

Reply:
Issue #199